mobile_verifier/src/verifier.rs

```diff
@@ -179,21 +249,24 @@ impl VerifierDaemon {
         let written_files = self.mobile_rewards.commit().await?.await??;
 
         let mut transaction = self.pool.begin().await?;
-        // Clear the heartbeats table:
-        sqlx::query("TRUNCATE TABLE heartbeats;")
+
+        // Clear the heartbeats of old heartbeats:
+        sqlx::query("DELETE FROM heartbeats WHERE truncated_timestamp < $3")
```

Comment:
should this be `$1`? will postgres automatically convert the input timestamp to a truncated timestamp (i.e. they are comparable?)

Reply:
Fixed 9962212